index.js

```diff
@@ -40,9 +40,24 @@ const {
   MER_ERR_METHOD_NOT_ALLOWED,
   MER_ERR_INVALID_METHOD
 } = require('./lib/errors')
-const { Hooks, assignLifeCycleHooksToContext, assignApplicationLifecycleHooksToContext } = require('./lib/hooks')
-const { kLoaders, kFactory, kSubscriptionFactory, kHooks } = require('./lib/symbols')
-const { preParsingHandler, preValidationHandler, preExecutionHandler, onResolutionHandler, onGatewayReplaceSchemaHandler } = require('./lib/handlers')
```

Comment:
@mcollina If I run `npm run lint:fix` nothing changes... in fact, the format changes you mention were made by the configured linter (standard). 🤷‍♂️

Reply:
> If I run npm run lint:fix nothing changes...

These lines pass linting. However they make reviewing this PR impossible.

> in fact, the format changes you mention were made by the configured linter (standard). 🤷‍♂️

This is not correct as running standard does not make those changes.

Reply:
I don't want to change anything outside the scope of this PR, indeed... maybe those lines changed because of my prettier extension for VS code, but, to be fair, by running `npm run lint:fix` it should undo those format changes... right? Anyways, if standard can't undo them I'll try to fix this manually. Sorry for the inconvenience.

Reply:
@mcollina I've fixed the formatting issues